at_onboarding_flutter/lib/screen/at_onboarding_home_screen.dart

```diff
@@ -410,7 +410,10 @@ class _AtOnboardingHomeScreenState extends State<AtOnboardingHomeScreen> {
       absorbing: loading,
       child: Scaffold(
         appBar: AppBar(
-          title: const Text('Setting up your account'),
+          title: const Text(
+            'Pair your AtSign',
+            style: TextStyle(color: Color.fromARGB(255, 0, 0, 0)),
```

Comment:
Don't set text color. Because we have a theme switch function and it will make text color constant when turn on dark mode.

Reply:
Done, will resolve it in different ticket